app/assets/javascripts/application.js

```diff
@@ -48,3 +48,6 @@
 //= require grids.js
 //= require short-code-forms/dist/short-code-prompts.js
 //= require wikis.js
+//= require swagger-ui.js
```

Comment:
These look OK in here, but can you use Bower to include the various swagger libraries by version number instead of adding them into our codebase? You can add them in bower.json, and they seem to be registered on Bower: https://bower.io/search/